client/src/components/overview/components/features.jsx

```diff
@@ -1,41 +1,58 @@
-/*eslint-env es6*/
 import React from 'react'
 
-class Features extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      value: '',
-      features: '',
-    };
-  }
+// class Features extends React.Component {
+//   constructor(props) {
+//     super(props);
+//     this.state = {
+//       value: '',
+//       features: '',
+//     };
+//   }
 
-  makeFeatures() {
-    let featuresArr = [];
-    let i = 0;
-    for (let feature of this.props.features) {
-      featuresArr.push(<div key={i} id="feature">{feature.feature} {feature.value}</div>)
-      i++;
-    }
-    return <div id="Features">{featuresArr}</div>
-  }
+//   makeFeatures() {
 
-  componentDidMount() {
-  }
+//     let featuresArr = [];
+//     let i = 0;
+//     for (let feature of this.props.features) {
+//       featuresArr.push(<div key={i} id="feature">{feature.feature} {feature.value}</div>)
+//       i++;
+//     }
+//     return <div id="Features">{featuresArr}</div>
+//   }
+
+//   render() {
+
+//     console.log('features', this.props.features)
+//     return (
+//       <div className="features">
+//         {this.makeFeatures()}
+//       <div/>
+//     )
+//   }
+// }
 
+// export default Features
 
-  render() {
-    return (
-      <>
-        <div>Features Here</div>
-        {this.makeFeatures()}
-      </>
-    )
+
+let Features = (props) => {
```

Comment:
I think this is the beginning of the functional component Features? You can use const here if that's the case.